### PR TITLE
fix: sync status updates

### PR DESCRIPTION
**Pull request by nathaliabj**
### Pre requirements:
- [ ] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: 

### Description:

- [x] There is still a long delay between clicking the refresh button and the spinner happening
- [x] The interval for checking the jobs endpoint is extreme, there are so many requests being fired! We can probably make the interval at least 10 times longer (see image of number of calls I see in the network monitor in the space of 10s)
- [x] “sync in progress” is never cleared! Even after the sync is successful
- [x] “sync successful” only shows if i stay on the explore page, if I navigate away I never see it.
- [x] If you combine the last two points, it looks like the sync never finishes if you hit refresh and then navigate to another page
- [x]  The /status endpoint is still being polled constantly, I don’t think we need that anymore

### Scope of the changes (select all that apply):
- [ ] Frontend  
- [ ] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

**Comment by rephus** [comment]
I replaced /status with just /job , and fixed disabled button delay 
Peek 2022-04-27 15-43

**Comment by rephus** [comment]
would it be possible to "round" average ? even if it is hardcoded image

**Comment by rephus** [comment]
I see 2 toasts now... :thinking: 
Peek 2022-04-28 08-44

File: packages/frontend/src/App.tsx [hunk 1]
```diff
@@ -244,6 +245,7 @@ const App = () => (
                                         </div>
                                     </PrivateRoute>
                                 </Switch>
+                                <RefreshStepsModal />
```
Comment (nathaliabj) [inline packages/frontend/src/App.tsx:248]: @owlas @rephus I have moved it to the app, so the modal can be open from everywhere in the app

**Review by rephus — COMMENTED** [review]
LGTM. Did you fix the issue when the browser lost focus ?

File: packages/frontend/src/App.tsx (hunk 1 above)
Comment (owlas) [reply packages/frontend/src/App.tsx:248]: Nice!!

File: packages/frontend/src/hooks/useRefreshServer.tsx [hunk 2]
```diff
@@ -1,45 +1,84 @@
...
-            if (data.jobStatus === 'DONE') {
-                showToastSuccess({
-                    title: `Sync successful!`,
-                });
-            }
-        },
-        onError: (result) => setErrorResponse(result),
-        refetchIntervalInBackground: false,
-        refetchOnMount: false,
+        refetchInterval: (data) => data?.jobStatus === 'RUNNING' && 1000,
```
Comment (owlas) [inline packages/frontend/src/hooks/useRefreshServer.tsx:80]: This interval could be 10 times larger

**Review by owlas — CHANGES_REQUESTED** [review]

**Pull request merged**